examples/file-upload.html

```diff
@@ -0,0 +1,181 @@
+<title>snuggsi ツ - File Upload</title>
+
+<meta charset=UTF-8>
+<meta name=viewport content=width=device-width,initial-scale=1>
+
+<link rel=stylesheet href=http://www.devpunks.com/typography.css>
+
+<script nomodule    src=https://unpkg.com/snuggsi/snuggsi.js></script>
+<script nomodule    src=https://unpkg.com/snuggsi/examples/webcomponents-hi-ce.js></script>
+
+<hgroup>
+  <h1>snuggsi ツ</h1>
+  <h2>File Upload</h2>
+</hgroup>
+
+<file-upload>
+
+  <input multiple id=files name=files[] type=file>
+  <label for=files ondragover="event.preventDefault()" ondragenter=ondragstart ondragleave=ondragend ondrop=ondragdrop>
+    Click to upload file...
+  </label>
+
+  <table style="width:300px">
+    <thead style="text-align: left">
+      <tr>
+        <th width="1%"></th>
+        <th>Name</th>
+        <th width="10%">Size</th>
+        <th width="1%"></th>
+      </tr>
+    </thead>
+    <tbody>
+      <template name=files>
+        <tr>
+
+          <td>
+            <img src={icon_url} alt={type} hidden={hidden} style="width: 1em; height: 1em">
```

Comment:
This is the image you showed me: 
![image](https://cloud.githubusercontent.com/assets/7538743/25449911/a9c9a34c-2a93-11e7-809c-1a7bbc6374e2.png)
You can see that if it's an empty string it does the trick! :stuck_out_tongue: